snapcraft/internal/remote_build/errors.py

```diff
@@ -126,3 +129,18 @@ class AcceptPublicUploadError(RemoteBuildBaseError):
         "Remote build needs explicit acknowledgement that data sent to build servers "
         "is public.\nIn non-interactive runs, please use option --accept-public-upload."
     )
+
+
+class LaunchpadGitPushError(SnapcraftException):
+    def __init__(self, *, command: str, exit_code: int) -> None:
+        self._command = command
+        self._exit_code = exit_code
+
+    def get_brief(self) -> str:
+        return "Failed to push sources to launchpad."
```

Comment:
add launchpad's url instead of just launchpad

Reply:
Uppercase launchpad to Launchpad

Reply:
Will fix case.

Reply:
Missing unit test update

Reply:
Drat, fixed!